rdrf/rdrf/templates/registration/activate_account.html

```diff
@@ -0,0 +1,12 @@
+{% extends "rdrf_cdes/base-1-col.html" %}
+{% load project_title %}
+{% load i18n %}
+
+{% block content %}
+    <div class="alert alert-primary" role="alert">
+        <h3>{% trans "Please activate your account." %}</h3>
+        <p>{% trans "You are required to activate your account to gain access to the " %}{% project_title %}</p>
```

Comment:
Maybe a blocktrans would have been better here to allow for a placeholder (project_title) that would provide better context for the string in Crowdin.